inc/system/requirement/protectedwebaccess.class.php

```diff
@@ -113,13 +113,15 @@ protected function check() {
             $this->validation_messages[] = __('Web access to files directory is protected');
          }
       } else {
-         $this->validated = false;
-         $this->validation_messages[] = __('Web access to the files directory should not be allowed but this cannot be checked automatically on this instance.');
-         $this->validation_messages[] = sprintf(
-            __('Make sure access to %s (%s) is forbidden; otherwise review .htaccess file and web server configuration.'),
-            __('error log file'),
-            $CFG_GLPI['root_doc'] . '/files/_log/php-errors.log'
-         );
+         if(!$CFG_GLPI["ssovariables_id"] > 0){
```

Comment:
Not sure, but I guess the problem here is that the index.php script redirects to an unreachable URL in some SSO configurations. Maybe the check that should validate that access to GLPI servier is OK should be done on `if ($fic = fopen($uri.'/front/index.php', 'r', false, $context)) {`. Indeed, as it is an empty script, it should not do any unexcpected error/redirection.

Reply:
The problem is rather for "SSO" configurations (NTLM for example) which do the same role as the "AuthType" feature on Apache: any server side "fopen" (without correct authentication) will return an HTTP/401 Unauthorized, and therefore the warning will be displayed.